pyomo/repn/plugins/cpxlp.py

```diff
@@ -447,8 +447,11 @@ def _print_model_LP(self,
         #                                        labeler)))
         all_blocks = list( model.block_data_objects(
                 active=True, sort=sortOrder) )
-        variable_list = list( model.component_data_objects(
-                Var, sort=sortOrder) )
+        variable_list = list(model.component_data_objects(
+            Var,
+            active=True,
```

Comment:
> Only look at active variables
> We are looping through all inactive variables as well

If we set a variable to be inactive, we would not expect it to be included in the model to be written. This will therefore exclude inactive variables and make `variable_list` potentially considerably smaller.

Reply:
I've reverted. Looks like their tests were specifically looking at finding inactive variables here (which I don't really understand...). More investigation required on my part at a later date.